src/iconsManifest/supportedExtensions.ts

```diff
@@ -1319,6 +1319,13 @@ export const extensions: IFileCollection = {
       light: true,
       format: FileFormat.svg,
     },
+    {
+      icon: 'denoify',
+      filenamesGlob: ['denoify.config'],
+	    extensionsGlob: ['json', 'js', 'cjs'],
```

Comment:
Just wanted to note this, as it'll get picked up by checks. Gotta keep consistent indents 👍 
```suggestion
      extensionsGlob: ['json', 'js', 'cjs'],
```

Reply:
Damn sorry about that.  
Good catch